atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -281,6 +282,7 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
         super(OrchestrateBuildPlugin, self).__init__(tasker, workflow)
         self.platforms = set(platforms)
         self.build_kwargs = build_kwargs
+        self.osbs_client_config_fallback = osbs_client_config
         self.osbs_client_config = osbs_client_config
```

Comment:
self.osbs_client_config is now a dead parameter that never gets used. Let's delete it.

Reply:
Good catch, for forgot to remove it after debugging. Fixed